src/plugins/intel_npu/src/plugin/src/compiled_model.cpp

```diff
@@ -128,17 +138,25 @@ std::shared_ptr<ov::ISyncInferRequest> CompiledModel::create_sync_infer_request(
 }
 
 void CompiledModel::export_model(std::ostream& stream) const {
-    const auto& blob = _networkPtr->compiledNetwork;
+    _logger.info("CompiledModel::export_model");
+
+    // FOr CID path mean it get the blob from compiler through pfnGetNativeBinary using
+    // networkDescription->metadata.graphHandle
+
+    // For CIP path mean it get the blob form
+    // networkDescription->compiledNetwork
+    std::vector<uint8_t> blob = _compiler->getCompiledNetwork(_networkPtr);
```

Comment:
```suggestion
    const auto& blob = _compiler->getCompiledNetwork(_networkPtr);
```

Reply:
ok, updated in new commit,
but using `const auto& blob`,  `const auto&& blob` might better to eliminate an extra copy